core/logger/logger_test.go

```diff
@@ -176,13 +201,53 @@ func TestWithFileRotation(t *testing.T) {
 	file := log.File()
 	assert.NotNil(t, file)
 }
+
 func TestWithStderrRedirect(t *testing.T) {
 	log := logger.DefaultLogger()
 
 	err := log.ApplyOptions(logger.WithStderrRedirect())
 	assert.Nil(t, err)
 }
 
+func TestHeadline(t *testing.T) {
+	log := logger.DefaultLogger()
+	message := "One blockchain to rule them all"
+	delimiter := "///"
+	timestamp := "-"
+
+	formatedMessage := log.Headline(message, delimiter, timestamp)
+	assert.Contains(t, formatedMessage, message)
+	assert.Contains(t, formatedMessage, delimiter)
+	assert.Contains(t, formatedMessage, timestamp)
+}
+
+func TestRedirectStderr(t *testing.T) {
+	file, _ := core.CreateFile("", "logs", "log")
+	err := logger.RedirectStderr(file)
+	assert.Nil(t, err)
+}
+
+func TestRedirectStderrWithNilFile(t *testing.T) {
+	err := logger.RedirectStderr(nil)
+	assert.NotNil(t, err)
+}
+
+func TestRollFiles(t *testing.T) {
```

Comment:
This test causes multiple tests to fail with the race flag set

Reply:
That was because some of the tests used DefaultLogger. Changed to ElrondLogger, and all tests are passing now.